docs/language/aggregates/README.md

```diff
@@ -0,0 +1,17 @@
+# Aggregate Functions
+
+Aggregate functions appear in either summarization
+or expression context and produce an aggregate value for a sequence of inputs values:
```

Comment:
I think this would read better with a period instead of a colon since it doesn't really introduce the list that follows. 
```suggestion
or expression context and produce an aggregate value for a sequence of inputs values.
```